acceptance/tests/api-gateway/api_gateway_gatewayclassconfig_test.go

```diff
@@ -144,10 +128,6 @@ func TestAPIGateway_GatewayClassConfig(t *testing.T) {
 	logger.Log(t, "creating controlled gateway")
 	gateway := createGateway(t, k8sClient, gatewayName, namespace, gatewayClassName, certificateName)
 
-	// make sure it exists
-	logger.Log(t, "checking that gateway one is synchronized to Consul")
-	checkConsulExists(t, consulClient, api.APIGateway, gatewayName)
-
```

Comment:
We combed through https://github.com/hashicorp/consul-k8s/pull/2481 and removed anything that looked like it was added accidentally.

The lines in this file were previously removed in https://github.com/hashicorp/consul-k8s/pull/2409